authority/provisioner/x5c.go

```diff
@@ -287,6 +292,11 @@ func (p *X5C) AuthorizeSSHSign(ctx context.Context, token string) ([]SignOption,
 		data.SetToken(v)
 	}
 
+	// The X509 certificate will be available using the template variable
+	// AuthorizationCrt. For example {{ .AuthorizationCrt.DNSNames }} can be
+	// used to get all the domains.
+	data.SetAuthorizationCertificate(claims.chains[0][0])
```

Comment:
`.AuthorizationCrt.DNSNames` is always `null` for certificates issued by OIDC provisioner (I don't know why), so I needed to rely on `.AuthorizationCrt.Subject.CommonName` (https://github.com/smallstep/docs/pull/125)

Reply:
Certificates issued by OIDC usually have email addresses, `{{ .AuthorizationCrt. EmailAddresses }}`.